go.mod

```diff
@@ -97,21 +101,23 @@ require (
 	github.com/mitchellh/go-linereader v0.0.0-20190213213312-1b945b3263eb
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
 	github.com/oracle/oci-go-sdk v5.7.0+incompatible
+	github.com/packethost/packngo v0.5.0
 	github.com/pascaldekloe/goe v0.1.0 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/client_model v0.2.0
+	github.com/rogpeppe/godef v1.1.2 // indirect
```

Comment:
I'm guessing this is a packethost indirection?

Reply:
This indirect is cleared with a `go mod tidy`, it may have been from a previous version of packngo that was imported.